web/src/app/services/ServiceDeleteDialog.js

```diff
@@ -23,7 +22,11 @@ function DeleteForm({ epName, error, value, onChange }) {
             value='delete-escalation-policy'
           />
         }
-        label={`Also delete escalation policy: ${epName}`}
+        label={
+          <React.Fragment>
+            `Also delete escalation policy: {epName}
```

Comment:
```suggestion
            Also delete escalation policy: {epName}
```

Reply:
Any benefit to change this to use a fragment instead of the 1 line we had before?

Reply:
Aside from the ``` ` ``` typo, this was needed so that `epName` could be text or a React node. Putting it in the string/line like before will render `[object Object]`

Reply:
Will `epName` ever not be a string?

Reply:
Yep, propTypes need to be updated to `p.node.isRequired` -- will do